nd4j/nd4j-backends/nd4j-backend-impls/nd4j-aurora/src/main/java/org/nd4j/linalg/cpu/nativecpu/CpuBackend.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *  ******************************************************************************
+ *  *
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Apache License, Version 2.0 which is available at
+ *  * https://www.apache.org/licenses/LICENSE-2.0.
+ *  *
+ *  *  See the NOTICE file distributed with this work for additional
+ *  *  information regarding copyright ownership.
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  * License for the specific language governing permissions and limitations
+ *  * under the License.
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *  *****************************************************************************
+ */
+
+
+package org.nd4j.linalg.cpu.nativecpu;
+
+import org.nd4j.linalg.factory.Environment;
+import org.nd4j.linalg.factory.Nd4jBackend;
+import org.nd4j.common.io.ClassPathResource;
+import org.nd4j.common.io.Resource;
+import org.nd4j.nativeblas.NativeOpsHolder;
+
+/**
+ * Cpu backend
+ *
+ * @author Adam Gibson
+ */
+public class CpuBackend extends Nd4jBackend {
```

Comment:
Does it make sense for the nativecpu package to exist in the aurora build?

Reply:
@treo fair enough..let me switch the namespace there.